client/scss/components/_footer.scss

```diff
@@ -166,3 +166,58 @@
     }
   }
 }
+
+// Footer control bar for performing actions on the page
+footer .actions,
+footer .preview {
+  .button {
+    @apply w-leading-none w-inline-flex w-items-center;
```

Comment:
What's the reasoning of using tailwind apply here? Should we be consistently not using apply?

Reply:
This is just moving existing code.